requirements/edx/base.in

```diff
@@ -171,21 +150,48 @@ social-auth-core
 simplejson
 Shapely                             # Geometry library, used for image click regions in capa
 six                                 # Utilities for supporting Python 2 & 3 in the same codebase
-skill-tagging                       # Library needed to support tagging of skills in unit/video xblocks via forms
 social-auth-app-django
 sorl-thumbnail
 sortedcontainers                    # Provides SortedKeyList, used for lists of XBlock assets
 sqlparse                            # Required by Django to run migrations.RunSQL
 stevedore                           # Support for runtime plugins, used for XBlocks and edx-platform Django app plugins
-staff-graded-xblock                 # An XBlock that allows for off-site bulk scoring of problems
-super-csv                           # Generic CSV processor
 unicodecsv                          # Easier support for CSV files with unicode text
 user-util                           # Functionality for retiring users (GDPR compliance)
 webob
 web-fragments                       # Provides the ability to render fragments of web pages
 XBlock[django]                      # Courseware component architecture
 xblock-utils                        # Provides utilities used by the Discussion XBlock
 xss-utils                           # https://github.com/edx/edx-platform/pull/20633 Fix XSS via Translations
+
+########################################################################################################################
+## Non-kernel dependencies
+# The following dependencies do not seem to be core to the platform's functionality, and will soon be moved to a
```

Comment:
I recommend actually breaking this file out as part of this PR (you can just add `-r default.in` to `base.in`, for now). My experience with the github.in cleanup is that folks don't always read comments before adding to a file, so the more that you can make it obvious that things are intentionally split, the more likely folks are to follow the rules you're establishing.

Reply:
I started that way, but the diff was harder to review with everything moved around to new files. Now that you've reviewed it, that's less of a problem so I can do that. Or I can do a fast-follow PR to move them to separate files which was my plan.

Reply:
I decided to do this now, and I also moved most of `base.in` to `kernel.in`. Keeping `base.in` in place for now should avoid breaking anything, and we can iterate from here. In the next PR I want to get the test suite passing without `bundled.in` installed.

Reply:
I like the idea of fast-follow to reduce risk on what you've done so far.

Reply:
Do you think what I just did in ed033aa197784612ad0e7f831f8df1f79e31af67 is risky? I tested `make upgrade` and it's working well. I think most things operate on base.txt so there should be no change, but maybe there's some tooling I'm not aware of. I can do that as a fast follow or leave it in this PR.

Reply:
I wrote my comment before seeing your change, and imagined `base.in` going away. I think what you did is fine. Thanks @bradenmacdonald.